xml/art_sle_ha_install_quick.xml

```diff
@@ -748,18 +774,16 @@ Full list of resources:
     </step>
     <step>
      <para>
-      Rerun the <command>crm status</command> command. You should see the floating
-      virtual IP address moved to <literal>&node2;</literal>:
+      Watch the uninterrupted flow of pings to your virtual IP address.
      </para>
-     <screen>&prompt.root;<command>crm</command> status
-[...]
-Full list of resources:
-
- vip    (ocf::heartbeat:IPaddr2):       Started &node2;</screen>
     </step>
     <step>
      <para>
-      Watch the uninterrupted flow of pings to your virtual IP address.
+      Look into &hawk2; <menuchoice>
```

Comment:
Maybe better: In &hawk2;, click <menuchoice><guimenu>Status</guimenu>
 <guimenu>Resources</guimenu> </menuchoice>. The resource <systemitem>admin_addr</systemitem>
should be running on the <systemitem>&node2;</systemitem> node now.